config/core/build-configs.yaml

```diff
@@ -190,6 +190,42 @@ fragments:
   x86_kvm_guest:
     path: "kernel/configs/kvm_guest.config"
 
+  ltp_crypto:
```

Comment:
Please rename to `crypto` as this is not necessarily for LTP.  It can be useful for any tests that require crypto algorithms to be enabled.